package.json

```diff
@@ -59,6 +59,8 @@
     "@types/slate-react": "0.22.5",
     "@types/testing-library__react-hooks": "^3.1.0",
     "@types/tinycolor2": "1.4.2",
+    "@typescript-eslint/eslint-plugin": "^2.18.0",
```

Comment:
```suggestion
    "@typescript-eslint/eslint-plugin": "2.18.0",
```

Reply:
Why when we have a lockfile?

Reply:
We had couple of issues in the past when regenerated yarn.lock(from scratch) was referencing versions which were causing troubles.